applications/visualizer/frontend/src/components/AppLauncher.tsx

```diff
@@ -109,12 +123,7 @@ function AppLauncher() {
                 About Nemanode
               </Button>
             </Box>
-            <Chip
-              icon={<BarChart />}
-              label={`${TEMPLATE_ACTIVE_DATASETS.length} datasets, ${neuronNames.length} neurons`}
-              variant="outlined"
-              className="basic"
-            />
+            <Chip icon={<BarChart />} label={`${dataSetsCount} datasets, ${neuronsCount} neurons`} variant="outlined" className="basic" />
```

Comment:
when fetching the datasets and neurons this will display first 0 and then the updated numbers. Could you render a dash (-) instead of zero when we are still fetching the numbers?

I will initialise datasetcount and neuronscount to undefined, when undefined we show the dash otherwise if an error happens in the request/fetch we set these counters to 0 and then we display the 0.
Thanks!